privacyidea/lib/eventhandler/usernotification.py

```diff
@@ -301,7 +306,8 @@ def do(self, action, options=None):
                     with open(filename, "r", encoding="utf-8") as f:
                         body = f.read()
                 except Exception as e:
-                    log.warning(u"Failed to read email template from file {0!r}: {1!r}".format(filename, e))
+                    log.warning(u"Failed to read email template from file "
```

Comment:
Are you breaking those lines manually? Or does your IDE break it automatically?
Since the line still is shorter than 120 characters. I think we confirmed on this line length.
If we confirm on a certain line length I would recommend to use this and not add breaks where not necessary

Reply:
I've added the breaks since i find it more readable. Also i would prefer a 100 chars line limit overall, but this can be discussed later.
I'll fix this to 100 chars for now.

Reply:
I think 100 chars is to short. And we added 120 chars in the .pep8speaks.
This line is 108 chars. I think there are a lot of lines that are longer than 100chars. Because an average notification text might be around 70-80 lines. With indentation and command such a line will almost always be longer than 100 lines. So you will break all those lines.
Please stick with 120.

Reply:
ok

Reply:
Done in bb386797